crates/astria-sequencer/src/mempool.rs

```diff
@@ -20,7 +20,9 @@ pub(crate) struct TransactionPriority {
 
 impl TransactionPriority {
     fn nonce_diff(&self) -> u32 {
```

Comment:
Not part of this PR, but could you move this type's constructor (`TransactionPriority::new`) into this impl block? First thing I did was look for it and then realized it's so far down.

Reply:
I have removed this constructor in #1092, so it won't be an issue soon :)